starcat/starcatalog.py

```
@@ -6,7 +6,7 @@
 
 import inspect
 import numpy as np
-from typing import Any, Iterator, Optional
+from typing import Any, Iterator, Optional, no_type_check
```

Ruff (0.14.4)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)